### ISSUE TITLE
Password reset not working in some cases

### ISSUE THREAD

**Issue by tim-antkowiak** [JWManagement/JWManagement#443]
For some users the password reset link is not working. It always aborts with the error message: "This link is invalid.
Please request another reset password mail."

Background: If the user requests a password reset, a random token is generated and an email with a link to the reset page including that token is sent to the user. The token is also saved to the user at "services.password.reset.token".
If the link is clicked the app searches for the user with that token and provides to update the password.

There are multiple things I noticed:
- The token in the link which is sent to the user is all lowercase (depending on mail client?), in the DB the token is saved as it is generated (mixed cases)
- Even if the above error message is displayed the token seems to get removed from the user in the DB
- The users with issues used their mail address as username

**Comment by aloosen** [comment]
Do we have any information about which clients are affected? None that I know change content from mixed case to lower case

**Comment by studiozandra** [comment]
Not sure if this helps: when I do a password reset, I get the "link invalid" message very briefly across the screen, but then it proceeds with the password reset process. Firefox on Windows 10 (also reproduced in Firefox on Linux and Android).

**Comment by aloosen** [comment]
@studiozandra Yes, I can reproduce that. When I switch my connection to GPRS speed, I can also see the "link invalid" page. Not sure if that's related to the original problem, though.

**Comment by kheinzel** [comment]
Is there any solution? I have the same problem and can't reset my password. Thank you.

**Comment by aui01** [comment]
Hello, actually there is no Fix for this issue available.But we can solve it manually. Please reply to this mail and tell me the name of the project. Best regards Alex | JW Management Support Am 26.08.2024 um 13:23 schrieb kheinzel ***@***.***>:﻿
Is there any solution? I have the same problem and can't reset my password. Thank you.

**Comment by kheinzel** [comment]
Thank you. Solved like this: We deleted my account and created a new one.
